keda/Chart.yaml

```diff
@@ -4,15 +4,15 @@ description: Event-based autoscaler for workloads on Kubernetes
 
 # Specify the Kubernetes version range that we support.
 # We allow pre-release versions for cloud-specific Kubernetes versions such as  v1.21.5-gke.1302 or v1.18.9-eks-d1db3c
-kubeVersion: ">=v1.24.0-0"
+kubeVersion: ">=v1.23.0-0"
```

Comment:
Why revert? We should just remove it imo. No point in enforcing 1.23 if we don't want to enforce things.

Reply:
well, KEDA won't work on anything below 1.23.0

Reply:
>= 1.23.0 is a technical reason

>= 1.24.0 or >= 1.25.0 is just a project management reason

🤷‍♂️

Reply:
It can give a wrong impression though but it's fine